my-project/src/Components/Execrise3.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+function ClickCounter() {
+  const [count, setCount] = useState(0);
+
+  const incrementCount = () => {
+    setCount(count + 1);
```

Comment:
Did you know you can access the 'count' value inside the 'setCount' function like this:
```suggestion
    setCount((prev) => prev + 1);
```